life_expectancy/cleaning.py

```diff
@@ -5,37 +5,68 @@
 
 desired_order = ['unit', 'sex', 'age', 'region', 'year', 'value']
 
+def load_data(file_path):

```

Comment:
You should add types to variables. 
**Example:** _def load_data(file_path: Path):_